File: papi/lista1/exercicio17.py
while True:
    try:
       temperatura_fahrenheit = float(input("Por favor digite a temperatura em fahrenheit: "))
       break
    except ValueError:
        print("Erro: Por favor digite um valor numerico válido, com o ponto como um separador decimal")

print("O equivalente em celsius é:", 5 * (temperatura_fahrenheit-32) / 9)

File: papi/lista1/exercicio17.kt
package lista1

fun main() {
    var temperaturaFarenheit: Double = 0.0 
	while (true) {
        try {
            print("Digite a temperatura em Farenheit: ")
            var input = readLine()
            temperaturaFarenheit = input!!.toDouble()
            break
        } catch (e: NumberFormatException) {
            println("Por favor, digite um valor numérico, com um ponto como separador decimal.")
        }
    }

    println("A temperatura em Farenheit digitada foi: $temperaturaFarenheit")
    val temperaturaCelsius = (5 * (temperaturaFarenheit-32) / 9)
    println("A temperatura equivalente em celsius é: $temperaturaCelsius")
        
}